surveys/2024-annual-survey/translations/de.md

```diff
@@ -311,13 +311,13 @@ Rows:
 - Variadische Generics
 - Beliebige Self-Typen
 - Enum Varianten Typen
-- Allokator trait, bessere OOM-Verwaltung
+- Allokator Trait, bessere OOM-Verwaltung
 - Stabile ABI
 - Portables SIMD
 
 Columns:
 
-- Würde es mir erlauben, Rust zu nutzen
+- Würde eine Blockade bei meinem Anwendungsfall lösen
```

Comment:
Part of the discussion in https://github.com/rust-lang/surveys/issues/320

Suggestions for better wording are welcome though.

Reply:
maybe it's me but I associate "Blockade" with political contexts

Reply:
Yeah, maybe "Hindernis ... beseitigen" instead?